paddle/fluid/operators/transpose_op.cu

```diff
@@ -667,6 +667,7 @@ class TransposeGPUKernel : public framework::OpKernel<T> {
     auto* x = context.Input<framework::Tensor>("X");
     auto* out = context.Output<framework::Tensor>("Out");
     out->mutable_data<T>(context.GetPlace());
+    if (out->numel() == 0) return;
```

Comment:
Write this style, same for other similar places:
```
if (out->numel() == 0) {
    return;
}
```

Reply:
Thanks. Done.